internal/usagegen/codesamples.go

```diff
@@ -27,7 +27,7 @@ func GenerateCodeSamplesOverlay(ctx context.Context, schema, header, token, conf
 			schema,
 			header,
 			token,
-			"",
+			"./speakeasyusagegen",
```

Comment:
this makes the code samples generate in another directory basically causing the cleanup to try and cleanup repo/speakeasyusagegen instead of repo